crates/polars-ops/src/chunked_array/datetime/replace_time_zone.rs

```diff
@@ -68,7 +68,10 @@ pub fn replace_time_zone(
         }),
     };
     let mut out = out?.into_datetime(datetime.time_unit(), time_zone.map(|x| x.to_string()));
-    if from_time_zone == "UTC" && ambiguous.len() == 1 && ambiguous.get(0).unwrap() == "raise" {
+    if from_time_zone == "UTC"
+        && ambiguous.len() == 1
+        && unsafe { ambiguous.get_unchecked(0) } == Some("raise")
```

Comment:
I don't think we should use `unsafe` here. This isn't called repeatedly in a hot loop.

Reply:
right, thanks